certif/app/styles/components/user-logged-menu.scss

```
@@ -54,8 +54,9 @@
   top: 60px;
   right: 0;
   min-width: 400px;
+  max-width: 600px;
   max-height: 600px;
-  overflow: scroll;
+  overflow: auto;
```

Dans ce cas particulier, je voulais aussi proposer 
```suggestion
  overflow: auto;
  overscroll-behaviour-y: contain;
```
ça permet d'éviter de scroll la fenêtre derrière tant qu'on est en train de scroll dans le menu. C'est un comportement agaçant dans le cas où l'on tombe involontairement dans une zone qui le fait, mais là l'utilisateur ouvre consciemment le menu de choix de centre de certif.

À voir aussi si les cas d'usages en prod font que quelqu'un a suffisamment de centres de certifs à sélectionner pour atteindre la `max-height: 600px`.

---

Vu avec le métier, ajout du overscroll-behavior: contain;